config/clusters/smithsonian/common.values.yaml

```diff
@@ -76,7 +75,6 @@ basehub:
           description: &profile_list_description "Start a container with at least a chosen share of capacity on a node of this type"
           slug: small
           default: true
-          allowed_teams: *allowed_github_orgs
```

Comment:
Its better if we just allow all users by default instead, because otherwise we need to make a few extra API calls to check for membership etc for no real reason.

Since https://github.com/2i2c-org/infrastructure/pull/3234#issuecomment-1749180174, removing `allowed_teams` makes all users logged in to the hub be able to see this like we want.